doc/source/whatsnew/v0.22.0.txt

```diff
@@ -41,10 +41,10 @@ Other API Changes
 ^^^^^^^^^^^^^^^^^
 
 - ``NaT`` division with :class:`datetime.timedelta` will now return ``NaN`` instead of raising (:issue:`17876`)
-- All-NaN levels in ``MultiIndex`` are now assigned float rather than object dtype, coherently with flat indexes (:issue:`17929`).
+- All-NaN levels in a ``MultiIndex`` are now assigned ``float`` rather than ``object`` dtype, promoting consistency with ``Index`` (:issue:`17929`).
 - :class:`Timestamp` will no longer silently ignore unused or invalid `tz` or `tzinfo` keyword arguments (:issue:`17690`)
 - :class:`Timestamp` will no longer silently ignore invalid `freq` arguments (:issue:`5168`)
```

Comment:
On the two lines above: tz, tzinfo, and freq should have two backticks instead of one.

Reply:
fixed